hikari/api/special_endpoints.py

```diff
@@ -939,13 +939,24 @@ class CommandBuilder(abc.ABC):
 
     __slots__: typing.Sequence[str] = ()
 
+    @property
+    @abc.abstractmethod
+    def name_localizations(self) -> undefined.UndefinedOr[typing.Mapping[str, str]]:
+        """Name localizations set for this command.
+
+        Returns
+        -------
+        hikari.undefined.UndefinedOr[typing.Mapping[str, str]]
+            The name localizations of this command
+        """
```

Comment:
```suggestion
        """Name localizations set for this command."""
```

Reply:
Its something that I have been pushing for for a while. They will be removed with #1118 anyways, just helps me not have to go around searching for them again